quintilesims/layer0
# Enable aws_ami data source for l0-setup

**What does this pull request do?**
Enables l0-setup to use terraform's[ aws_ami data source](https://www.terraform.io/docs/providers/aws/d/ami.html#description).


**How should this be tested?**
Test by running `l0-setup init` and `apply`. The correct AMI for your region should be interpolated. 

Testing Caveat: you'll need to select the correct branch of the layer0 terraform module when running l0-setup `terraform_ami_update`. Unfortunately, selecting this branch as the module source when prompted by `l0-setup` seems to be broken. I was able to get around this by having `terraform get` pull down `master`, and then I manually switched branches in `~/.layer0/.terraform/...`.


**Checklist**
